target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -49,6 +49,9 @@ antrouter-r1)
 	ucidef_set_led_wlan "wlan" "WLAN" "$board:green:wlan" "phy0tpt"
 	ucidef_set_led_default "btc" "BTC" "$board:green:btc" "0"
 	;;
+ap531b0)
+        ucidef_set_led_wlan "wlan" "WLAN" "$board:green:wlan" "phy0tpt"
```

Comment:
This entry is already used by sc1750|\ and sc450) so You can move sc boards to your entry and delete those.